ansible_collections/arista/avd/roles/eos_validate_state/ANTA-Preview.md

```diff
@@ -66,6 +69,14 @@ title: Ansible Collection Role eos_valudate_state - Preview Integration with ANT
   '
   ```
 
+  To upgrade the package:
+
+  ```shell
+  pip3 install -U '
```

Comment:
I don't think you need -U when you assign the version with ==

Reply:
Oh right I didn't know that! Thanks